Sources/Hummingbird/Middleware/MetricsMiddleware.swift

```diff
@@ -49,17 +49,17 @@ public struct MetricsMiddleware<Context: RequestContext>: RouterMiddleware {
             // Don't record uri in 404 errors, to avoid spamming of metrics
             if let endpointPath = context.endpointPath {
                 dimensions = [
-                    ("hb_uri", endpointPath),
-                    ("hb_method", request.method.rawValue),
+                    ("uri", endpointPath),
```

Comment:
since everything is prefix with hb, I suppose we the next few lines should be prefix with `hb.` as well?

Reply:
Should metric metadata also include the prefix, the counter/timer it is attached to already has the prefix?

Reply:
Using Otel names